src/frontend/xgboost.cc

```diff
@@ -378,6 +378,21 @@ inline std::unique_ptr<treelite::Model> ParseStream(dmlc::Stream* fi) {
     xgb_trees_.back().Load(fp.get());
   }
   CHECK_EQ(gbm_param_.num_roots, 1) << "multi-root trees not supported";
+  // tree_info is currently unused.
```

Comment:
Out of curiosity, treelite doesn't have multi-class model?

Reply:
Do you mean multi-class classifiers? Treelite supports two methods for representing multi-class classifiers. See the docstring in https://github.com/dmlc/treelite/blob/49c35a52599d6728bf18552acddd5ee64fab419c/include/treelite/tree.h#L103-L121